packages/calypso-apps-builder/index.js

```diff
@@ -124,7 +124,7 @@ function setupRemoteSync( localPath, remotePath, shouldWatch = false ) {
 		} );
 
 		if ( shouldWatch ) {
-			chokidar.watch( localPath ).on( 'change', debouncedSync );
+			chokidar.watch( localPath ).on( 'all', debouncedSync );
```

Comment:
For some reason, `chokidar` has stopped issuing `change` events when files are created. `all` seems to work, though, so we use that instead.